### PR TITLE
Update examples to reflect changes to adafruit_lsm6ds

### PR DESCRIPTION
**Pull request by Breefield**
`adafruit_lsm6ds` is a module now, these imports must change to reflect that

### CODE REVIEW

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
@Breefield thanks for fixing these! Can you make the sub-module names lowercase so that they'll match the name of the file that that they're inside of. 
like: 
```python
from adafruit_lsm6ds.lsm6dox import LSM6DSOX as LSM6DS
```
Other than that these changes look good to me. Tested successfully on Feather nrf52840.

**Comment by Breefield** [comment]
👍

File: examples/lis3mdl_lsm6ds_test.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import time
 import board
-from adafruit_lsm6ds import LSM6DSOX as LSM6DS
+from adafruit_lsm6ds.lsm6Dsox import LSM6DSOX as LSM6DS
```
Comment (FoamyGuy) [inline examples/lis3mdl_lsm6ds_test.py]: One more capital `D` snuck in here.

**Review by FoamyGuy — APPROVED** [review]
Looks good to me. Thanks again @Breefield

**Pull request merged**